### ISSUE TITLE
Unraid 7 Support

### ISSUE THREAD

**Issue by Lucict** [stl88083365/unraid-nvidia-vgpu-driver#33]
Hello, 

Just wondering if this script can be used in Unraid 7 or if there are plans to support Unraid 7.  Thanks!

**Comment by llambkin** [comment]
Yeah, wanting to know also. Just found out after upgrading the plugin no longer works :/

**Comment by stl88083365** [comment]
可以使用
